.gitignore

```diff
@@ -22,6 +22,7 @@ my_narrative_venv/
 narrative_venv/
 new_narrative_env_2/
 js-coverage
+cover
```

Comment:
Added this because when I run make test, the new coverage files kept being picked up by git